File: ChartsDemo/Classes/RealmBase/RealmFloat.m
//
//  RealmFloat.m
//  ChartsDemo
//  Copyright © 2015 dcg. All rights reserved.
//

#import "RealmFloat.h"

@implementation RealmFloat

- (id)initWithFloatValue:(float)value
{
    self = [super init];
    
    if (self)
    {
        self.floatValue = value;
    }
    
    return self;
}

@end

File: ChartsDemo-OSX/ChartsDemo-OSX/RealmBase/RealmFloat.swift
//
//  RealmFloat.swift
//  ChartsDemo-OSX
//
//  Created by thierryH24A on 02/04/2017.
//  Copyright © 2017 dcg. All rights reserved.
//

import Cocoa
import Realm

class RealmFloat: RLMObject
{
    dynamic var floatValue: Float = 0.0
    
    override init() {
        super.init()
    }

    init(floatValue value: Float)
    {
        super.init()
        self.floatValue = value
    }
}
